framework/py/flwr/common/inflatable_grpc_utils.py

```diff
@@ -46,40 +47,51 @@
 
 
 def push_object_to_servicer(
-    obj: InflatableObject, stub: Union[FleetStub, ServerAppIoStub]
+    obj: InflatableObject,
+    stub: Union[FleetStub, ServerAppIoStub],
+    node: Node,
+    object_ids_to_push: Optional[list[str]] = None,
```

Comment:
Since we already use `set[str]` for the return type of `get_desdendant_object_ids`, I think we can also use `set` here. Wdyt?

```suggestion
    object_ids_to_push: Optional[set[str]] = None,
```